src/gui/resource.h

```diff
@@ -1,84 +1,87 @@
 //resource.h
 #pragma once
 
-//null resource - test
-#define IDR_NULL 0x0000
+enum {
 
-//fonts
-#define IDR_FNT_SMALL    0x0001
-#define IDR_FNT_NORMAL   0x0002
-#define IDR_FNT_BIG      0x0003
-#define IDR_FNT_TERMINAL 0x0004
-#define IDR_FNT_SPECIAL  0x0005
+    //null resource - test
+    IDR_NULL = 0x0000,
 
-//pngs
-#define IDR_PNG_splash_logo_marlin    0x0006
-#define IDR_PNG_splash_logo_prusa_prn 0x0007
-#define IDR_PNG_status_logo_prusa_prn 0x0008
+    //fonts
+    IDR_FNT_SMALL,
```

Comment:
I like this - here the enum perfectly solves the uniqueness of the resource ID's